CesiumIonClient/src/Connection.cpp

```diff
@@ -1,10 +1,15 @@
 #include "CesiumIonClient/Connection.h"
 
+#include "CesiumGeospatial/BoundingRegion.h"
+#include "CesiumGeospatial/Cartographic.h"
+#include "CesiumGeospatial/GlobeRectangle.h"
+#include "CesiumIonClient/Geocoder.h"
```

Comment:
This file is inconsistent to begin with, but we should move toward `#include <CesiumHeader/Whatever.h>` instead of `#include "CesiumHeader/Whatever.h"`.